dp-cluster-setup-utility.py

```diff
@@ -423,6 +423,17 @@ def enable_trusted_proxy_for_atlas(self):
       'atlas.proxyuser.%s.groups' % knox_user: '*',
     }, note='updated by dp-cluster-setup-utility')
 
+  def enable_trusted_proxy_for_beacon(self):
+    if not self.cluster.has_service('BEACON'):
```

Comment:
This check seems spurious, since prior checks for DLM must confirm that BEACON is installed.

Reply:
Yes. Will remove this check.